Pidgin.sln

```diff
@@ -115,4 +112,10 @@ Global
 		{746634A6-796C-4314-88E5-0DF026618AE6}.Release|x86.ActiveCfg = Release|Any CPU
 		{746634A6-796C-4314-88E5-0DF026618AE6}.Release|x86.Build.0 = Release|Any CPU
 	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {B9334E6E-D684-452D-839F-BAE7F16459EA}
+	EndGlobalSection
```

Comment:
Please could you revert the sln file changes?

Reply:
Yes, my apologizes.